Documentation/Getting Started with Open Disclosure.md

```diff
@@ -0,0 +1,85 @@
+# Getting started with Open Disclosure
+
+This is a list of resources used by Open Oakland's **_[Open Disclosure]_** team + instructions to build the site on your own computer. The core technologies we use in the project are [Ruby], [Jekyll] (a Ruby/Liquid-based static site CMS), [CircleCI], [Node.js], a smidge of Python, and a teeny bit of React. *Don't worry* if you don't know things on the list (or if you've never even written a line of code) because we try to be inclusive and there are always tasks to be [re]done regardless of your technical background. Our data comes from [public-sources] and is augmented by a [shared google sheet] we use for manually aggregating information other references. For more information about "how the site works," see the [front-end README], [the site's FAQ], or our [campaign-finance wiki].
+
+* **Our Website**: https://www.opendisclosure.io
+* **GitHub Repositories**:
+  * **Front-End Repo**: https://github.com/caciviclab/odca-jekyll
+  * **Back-End Repo**: https://github.com/caciviclab/disclosure-backend-static
+
+## Communication Resources
+* **Slack**: Join the Open Oakland Slack at `openoakland.slack.com`, then join our specific `#open-disclosure` channel
+* **Waffle Board** (Kaban/Trello-style task-board): https://waffle.io/caciviclab/disclosure-backend
+
+## People To Know?
+Lots of people have worked on Open Disclosure in its 2+ years of existance (thanks for joining!), but if you *really* want to know "the deal" try these folks (@slack-Ids from openoakland.slack.com)
+
+* _Project Leads_: Tom Dooner(@tdooner) Aaron Borden(@adborden)
+* _Designer_: :warning::warning::warning:
+* _City Liason_: :warning::warning::warning:
+* ???: :warning::warning::warning:
+*
+*
+
+---
+
+# How to build the site on your machine
+
+Clone _*both*_ the back and front-end repos into the same parent directory. :warning: The front-end ***requires*** that the back-end directory be a sibling _specifically named_ `..\disclosure-backend-static` :warning: in order to build as it is hardcoded that way in the gulpfile.
+
+Dependencies you should already have installed: [Ruby **2.3.4**], [Bundler],
+[Make],
+[Node 8.9+],
+[ImageMagick] (Only need for production builds?? :warning: ?? **TODO** ?? :warning:),Python ?, [PostgreSQL] (only needed for back-end builds??)
+
+## ** Build the Front-End **
+(You did see the warning above that you'll need both repositories, right?)
+
+1. **1st-time only**: `cd` into the front-end directory, run `make setup`, then run `make build`.
+
+2. **Update** your local data resources with `make pull-finance`
```

Comment:
I am going through these steps for the first time. It seems like you must run `make pull-finance` before you run `make build`.

If you try to run the build first, you see the following error:

```
Configuration file: /Users/stevenlybeck/Documents/projects/opendisclosure/odca-jekyll/_config.yml
            Source: /Users/stevenlybeck/Documents/projects/opendisclosure/odca-jekyll
       Destination: /Users/stevenlybeck/Documents/projects/opendisclosure/odca-jekyll/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
  Liquid Exception: Could not find document '_office_elections/oakland/2018-11-06/mayor.md' in tag 'link'. Make sure the document exists and the path is correct. in index.html
jekyll 3.8.3 | Error:  Could not find document '_office_elections/oakland/2018-11-06/mayor.md' in tag 'link'.
```